src/main/scala/introprog/IO.scala

```diff
@@ -127,4 +127,35 @@ object IO {
     import java.nio.file.{Files, Paths, StandardCopyOption}
     Files.move(Paths.get(from), Paths.get(to), StandardCopyOption.REPLACE_EXISTING)
   }
+
+    /**
+    * DANGER: silently deletes `fileName`.
+    *
+    * @param fileName the path the file that will be deleted.
+    * */
+  def delete(fileName: String): Unit = {
+    import java.nio.file.{Files, Paths, StandardCopyOption}
```

Comment:
Don't need to import StandardCopyOption as not used. Also "DANGER: silently" should be removed as deleting is the sole obvious purpose of this method (whereas move is different as delete there might be a hidden side effect).